src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
It's obviously working, but... `supportscoroutines` hurts something deep inside. And adding more `supports*` like that just makes it worse. And those names also get bulky.

While `mudlet` may be defined in mapping scripts, those are not always guaranteed to be there already. I don't think that makes it more ok than using `mudlet.supports`.

If I was a script author using coroutined who cared about users before 3.2, I'd probably start my script with something like
```lua
mudlet = mudlet or {}
local supports = mudlet.supports or {} --better localize table
-- ...
```